marketplace/deployer_util/expand_config.py

```diff
@@ -70,12 +73,16 @@ def expand(values_dict, schema, app_uid=''):
   for k, prop in schema.properties.iteritems():
     v = values_dict.get(k, None)
 
+    # The value is not delivered to the framework, so it can be filled by the framework.
```

Comment:
Thanks for adding the comment! I would suggest the following wording:

```
# The value is not explicitly specified and thus is eligible for auto-generation.
```

Reply:
Done.